pkg/build/apis/build/internal_helpers.go

```diff
@@ -0,0 +1,161 @@
+package build
```

Comment:
You may not add functions to this package.

Reply:
> You may not add functions to this package.

Our API types should just be types.  Create a package: `imageapihelpers` in `pkg/image/apiserver/` instead

Reply:
or even `pkg/build/apis/build/buildapihelpers`, but not here.  it allows super naughty things

Reply:
I like to be explicit... `pkg/build/apis/build/internal_helpers` :-)